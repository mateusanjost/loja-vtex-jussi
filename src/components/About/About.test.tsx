import TestRenderer from "react-test-renderer";
import AboutComponent from "./About";

describe("About", () => {
  it("should match snapshot", () => {
    const testRenderer = TestRenderer.create(<AboutComponent />);
    const jsonComponent = testRenderer.toJSON();

    expect(jsonComponent).toMatchSnapshot();
  });
});
