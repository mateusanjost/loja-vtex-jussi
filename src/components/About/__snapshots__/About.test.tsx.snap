// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match snapshot 1`] = `
<div
  className="sc-hKFxyN sc-eCApnc okUlG dcdZif"
>
  <div
    className="sc-hKFxyN sc-iCoGMd okUlG jmwWuu"
  >
    <div
      className="sc-jSFjdj ibkeYF"
    >
      OLÁ, SOMOS 
      <br />
      A JÜSSI
    </div>
    <p
      className="sc-gKAaRy jdFlYA"
    >
      A Jüssi é uma agência integrante do grupo global WPP que vem há 10 anos consolidando o pensamento voltado para produtos e resolução de problemas. Nosso área dedicada exclusivamente para Produtos Digitais é organizada em 6 especialidades: Product Managamenet, User Experience Design, SEO, Tecnologia, Agile e User Behavior Analytics.
    </p>
  </div>
  <img
    alt="Jussi"
    src="image-jussi2x.jpg"
  />
  <div
    className="sc-pNWdM euMegm"
  />
</div>
`;
